libethcore/EVMSchedule.h

```diff
@@ -29,6 +29,7 @@ struct EVMSchedule
 {
     EVMSchedule(): tierStepGas(std::array<unsigned, 8>{{0, 2, 3, 5, 8, 10, 20, 0}}) {}
     EVMSchedule(bool _efcd, bool _hdc, unsigned const& _txCreateGas): exceptionalFailedCodeDeposit(_efcd), haveDelegateCall(_hdc), tierStepGas(std::array<unsigned, 8>{{0, 2, 3, 5, 8, 10, 20, 0}}), txCreateGas(_txCreateGas) {}
+    int version = 0;
```

Comment:
Should this be unsigned since values are only >= 0?

Reply:
Well in general it's not a good idea to try to enforce non-negative values with `unsigned` (because it doesn't really enforce it)
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines.html#es106-dont-try-to-avoid-negative-values-by-using-unsigned
But now I think that I'll change it since other values in this struct are `unsigned` (gas prices), and in other parts of the code I made it `u256`

Maybe makes sense to make it `u256` here as well, but for now `unsigned` definitely will work, too.

Reply:
I was thinking less of enforcing non-negative values and more of code clarity, i.e. someone reading the code will be able to tell that version should be non-negative.